uvloop/loop.pyx

```diff
@@ -1,4 +1,4 @@
-# cython: language_level=3
+# cython: language_level=3, embedsignature=True
```

Comment:
Does `embedsignature`work for you? I couldn't make `inspect.signature` produce anything meaningful for `Loop` methods :(